### ISSUE TITLE
Define minimal schema/catalog for content containing physical product accessibility criteria

### ISSUE THREAD

**Issue by dmundra** [GSA/open-product-accessibility-template#3]

**Comment by dmundra** [comment]
Reviewing https://github.com/GSA/open-product-accessibility-template/blob/main/data/VPAT2.4/VPAT2.4Rev508--February2020.md, I think the following won't be needed in an OPAT:

* Evaluation Methods Used: This will be implied by selecting a catalog
* Applicable Standards/Guidelines: Same
* Terms: This will be forced by the schema.

**Comment by dmundra** [comment]
For the catalog, I spent a little bit of time generating a schema for the existing WCAG Json and thinking about how a librarian function would work. I found this https://stackoverflow.com/questions/7341537/tool-to-generate-json-schema-from-json-data and tried the https://github.com/Nijikokun/generate-schema that was able to take the JSON from https://github.com/w3c/wai-wcag-quickref/tree/gh-pages/_data and generate a schema but didn't validate with ajv.

I will add the wcag JSONs to the repo so we can use them for the catalog.

**Comment by dmundra** [comment]
For the minimal schema, I will submit a PR but here is my example that takes 'Table 1' and our technical notes:

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "$id": "opat.schema.json",
  "title": "OPAT",
  "description": "Open Product Accessibility Template (OPAT) as a YAML file.",
  "type": "object",
  "properties": {
    "title": {
      "description": "The title of the report in the heading format of '[Company Name] Accessibility Conformance Report'.",
      "type": "string"
    },
    "name-product-version": {
      "description": "Product name and version if available.",
      "type": "string"
    },
    "product-description": {
      "description": "A brief description of the product.",
      "type": "string"
    },
    "contact-information": {
      "description": "Primary contact email.",
      "type": "string"
    },
    "notes": {
      "description": "Any details or explanation about the product or the report.",
      "type": "string"
    },
    "chapters": {
      "type": "array",
      "items": {
        "type": "object",
        "properties": {
          "num": {
            "description": "Table or chapter number.",
            "type": "string"
          },
          "criteria": {
            "type": "array",
            "items": {
              "type": "object",
              "properties": {
                "num": {
                  "description": "Criteria number.",
                  "type": "string"
                },
                "components": {
                  "type": "array",
                  "items": {
                    "type": "object",
                    "properties": {
                      "name": {
                        "description": "Name of component. E.g. Web, Electronic Docs, Authoring Tools, Software, and so on.",
                        "type": "string"
                      },
                      "adherence": {
                        "type": "object",
                        "properties": {
                          "level": {
                            "description": "Conformance Level. Supports: The functionality of the product has at least one method that meets the criterion without known defects or meets with equivalent facilitation.\nPartially Supports: Some functionality of the product does not meet the criterion.\nDoes Not Support: The majority of product functionality does not meet the criterion.\nNot Applicable: The criterion is not relevant to the product.\nNot Evaluated: The product has not been evaluated against the criterion. This can be used only in WCAG 2.0 Level AAA.",
                            "enum": [
                              "supports",
                              "partially-supports",
                              "does-not-support",
                              "not-applicable",
                              "not-evaluated"
                            ]
                          },
                          "notes": {
                            "description": "Remarks and Explanations.",
                            "type": "string"
                          }
                        },
                        "required": ["level"]
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  },
  "required": [
    "title",
    "name-product-version",
    "product-description",
    "contact-information"
  ]
}
```

**Comment by dmundra** [comment]
Updated the schema based on feedback on the pull request.

**Comment by dmundra** [comment]
After the PR merges, provide links to schema and YAML example.

**Comment by dmundra** [comment]
Schema example from Sprint 2: https://github.com/GSA/open-product-accessibility-template/blob/6843b7a5cdd1328e1ba31c5eec50af826ed5c743/schema/opat-0.1.0.json
